### PR TITLE
[IMP][10.0] customer_outstanding_statement

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Minor changes proposal

### CODE REVIEW

File: customer_outstanding_statement/report/customer_outstanding_statement.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline customer_outstanding_statement/report/customer_outstanding_statement.py:2]: Please, don't delete copyright history if not strictly necessary.

File: customer_outstanding_statement/__manifest__.py [hunk 2]
```diff
@@ -12,7 +12,8 @@
     'website': 'https://github.com/OCA/account-financial-reporting',
     'license': 'AGPL-3',
     'depends': [
-        'account',
+        'account_invoicing',
+        'web',
```
Comment (MiquelRForgeFlow) [inline customer_outstanding_statement/__manifest__.py:16]: Don't put web. web module is installed with base.

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]
I think this PR is not for v10 but for v11.
Thus, it's a migration and then it should be done as all migrations:
 - [11.0][MIG] ...
 - Maintaining commit history. See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0.

File: customer_outstanding_statement/report/customer_outstanding_statement.py (hunk 1 above)
Comment (JordiBForgeFlow) [reply customer_outstanding_statement/report/customer_outstanding_statement.py:2]: remove # -*- coding: utf-8 -*- everywhere

File: customer_outstanding_statement/__manifest__.py [hunk 3]
```diff
@@ -12,7 +12,8 @@
     'website': 'https://github.com/OCA/account-financial-reporting',
     'license': 'AGPL-3',
     'depends': [
-        'account',
+        'account_invoicing',
```
Comment (MiquelRForgeFlow) [inline customer_outstanding_statement/__manifest__.py:15]: Also don't change account module to account_invoicing if strictly necessary.

This module is modifying things from account, not from account_invoicing.

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Pull request closed without merging**